grails-app/domain/au/org/emii/aatams/Sensor.groovy

```diff
@@ -92,6 +94,23 @@ class Sensor implements Embargoable {
         return null
     }
 
+    static Map<String, List<String>> validTransmitterMap = new HashMap<String, List<String>>() {{
+        def transmitterTypes1 = Arrays.asList("PINGER", "RANGE TEST");
```

Comment:
Strongly agree that these database values should not be hardcoded in the code if possible. Should be externally configured in config or database.